recipes/conda-libmamba-solver/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "conda-libmamba-solver" %}
+{% set version = "22.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda-incubator/conda-libmamba-solver/archive/{{ version }}.tar.gz  # temporary
+  sha256: 0b6121974b8ffac0437652275fde94818164408ecf99d02b6ea9346622bb95c7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - flit-core >=3.2,<4
+  run:
+    - python >=3.6
+    - libmambapy >=0.22
+    - conda >=4.12
```

Comment:
Shouldn't this be a `run_constrained` requirement?
Otherwise how can `conda` depend on this package given that it would introduce a cyclic dependency graph? `conda->conda_libmamba_solver->conda`

Reply:
I don't think Conda depends on it. AIUI this is a optional (soft) dependency of Conda. When around and the new solver is enabled, this will be used, but it won't be used by default (and can't be used when not installed).

Reply:
Yes, that's right, conda doesn't depend on it, at least while we're doing the "experiment" to get real world feedback for a while. The switch to depending on it in conda could happen in the future, but we'll be pretty clear about it when that happens. But it's more likely that we'll basically only install it in miniconda by default in the future as long as the "classic" solver is still around.

All of that isn't decided yet since we first need to make sure the libmamba solver (and the extra code around it) behave like the classic solver.